templates/web/login/layout/base-layout.html.twig

```diff
@@ -39,6 +39,7 @@
                 <script type="text/javascript" src="/assets/vendors/json2/json2{% if not app.debug %}.min{% endif %}.js"></script>
                 <![endif]-->
                 <script type="text/javascript" src="/assets/vendors/modernizr/modernizr{% if not app.debug %}.min{% endif %}.js"></script>
+                <script src="https://www.google.com/recaptcha/api.js" async defer></script>
```

Comment:
The library should installed via Bower or NPM and deployed in public assets instead of adding it directly from a CDN